dev/src/dev.clj

```diff
@@ -22,3 +37,19 @@
   (doseq [ns-name ns-names]
     (require ns-name :reload))
   (expectations/run-tests ns-names))
+
+(defmacro require-model
+  "Rather than requiring all models inn the ns declaration, make it easy to require the ones you need for your current session"
+  [model-sym]
+  `(require [(symbol (str "metabase.models." (quote ~model-sym))) :as (quote ~model-sym)]))
```

Comment:
you don't need this Toucan will resolve the model automatically if you quote it e.g.

```clj
(db/select 'User :id 100)
```